onboarding/src/Components/UserEdit/EmployeeEditPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import EmployeeEditForm from "./UserEditForm";
```

Comment:
Uncommon renaming import,
`[ES6 Feature]` allows, but is it really necessary? :confused: